Source/Core/Core/PowerPC/Jit64/JitAsm.cpp

```diff
@@ -94,9 +94,6 @@ void Jit64AsmRoutineManager::Generate()
     MOV(64, R(RSCRATCH), ImmPtr(CPU::GetStatePtr()));
     TEST(32, MatR(RSCRATCH), Imm32(static_cast<u32>(CPU::State::Stepping)));
     FixupBranch notStepping = J_CC(CC_Z);
-    ABI_PushRegistersAndAdjustStack({}, 0);
-    ABI_CallFunction(PowerPC::CheckBreakPoints);
-    ABI_PopRegistersAndAdjustStack({}, 0);
     MOV(64, R(RSCRATCH), ImmPtr(CPU::GetStatePtr()));
```

Comment:
In fact, this instruction is probably redundant now, too.

Reply:
The more I think about it, isn't this entire branch now useless?

Reply:
Indeed. I've replaced it with the same simple "should we exit" check as in `do_timing`.